src/vhdl/mega65r5.xdc

```diff
@@ -124,7 +124,7 @@ set_property -dict {PACKAGE_PIN C13 IOSTANDARD LVCMOS33} [get_ports kb_io2]
 set_property -dict {PACKAGE_PIN B13 IOSTANDARD LVCMOS33} [get_ports kb_jtagen]
 # Place Keyboard close to I/O pins
 create_pblock pblock_kbd0
-add_cells_to_pblock pblock_kbd0 [get_cells [list kbd0]]
+add_cells_to_pblock pblock_kbd0 [get_cells [list kbd0] [list mk2]]
```

Comment:
You add this to both r4 and r5 xdc, but not to r3. Any reason for that?

Reply:
There was no placement constraint in the r3 xdc.
I didn't bother to add it.